doc/sphinx_ext.rst

```diff
@@ -0,0 +1,19 @@
+Sphinx Extension
+================
+Some of Gossip depended packages uses hooks as part of their public API. As such, adding the hooks to the documentation is needed.
+Gossip has sphinx exntension implementation. For using it:
```

Comment:
Gossip supports doing just that through its sphinx extension. In order to use it, install gossip with...